semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -165,7 +168,7 @@ trait TextDocumentOps { self: SemanticdbOps =>
               // https://github.com/scalameta/scalameta/issues/665
               // Instead of crashing with "unsupported file", we ignore these cases.
               if (gsym0 == null) return
-              if (gsym0.isUseless) return
+              if (gsym0.isUselessOccurrence) return
```

Comment:
If this method is used only once, let's inline it? That'll reduce the conceptual surface of SymbolOps and further highlight domain-specificity of this logic.